jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/AbstractRuntimeManager.java

```diff
@@ -129,6 +131,9 @@ public AbstractRuntimeManager(RuntimeEnvironment environment, String identifier)
         if (eagerInit != null) {
         	engineInitEager = Boolean.parseBoolean(eagerInit);
         }
+        if (engineInitEager) {
```

Comment:
Im reusing existing variable, but maybe is better to use a new one, wdyt?

Reply:
ok. There is a problem. Nobody uses that flag and that one is a bit dangerous. Better a new one.

Reply:
ok changed